scapy/contrib/coap.uts

```diff
@@ -47,3 +47,15 @@ s = '\x16\x33\xa0\xa4\x00\x78\xfe\x8b\x60\x45\xd9\xe1\xc1\x28\xff\x3c\x2f\x3e\x3
     '\x74\x65\x72\x6e\x61\x6c\x20\x43\x6c\x6f\x63\x6b\x22\x3b\x63\x74\x3d\x30\x3b\x6f\x62\x73\x2c\x3c\x2f\x61\x73\x79' \
     '\x6e\x63\x3e\x3b\x63\x74\x3d\x30'
 CoAP in UDP(s)
+
+= building with a text/plain payload
+p = CoAP(ver = 1, type = 0, code = 0x42, msg_id = 0xface, options=[("Content-Format", "\x00")], paymark = "\xff")
+p /= Raw("\xde\xad\xbe\xef")
+str(p) == '\x40\x42\xfa\xce\xc1\x00\xff\xde\xad\xbe\xef'
+
+= dissection with a text/plain payload
+p = CoAP(str(p))
+p.ver == 1
```

Comment:
Be careful, only the last line of a test is checked. You should use `assert` statements.